About.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type ="text/css" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css">
+    <link rel="stylesheet" type="text/css" href="Styles.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>James | About</title>
+</head>
+<body>
+    <header>
+        <a href="Home.html" class="Homebutton">James</a>
+        <nav class="topnav">
+            <a href="About.html">About</a>
+            <a href="Portfolio.html">Portfolio</a>
+            <a href="Contact.html">Contact</a>
+        </nav>
+        <nav class="sociallinks">
+            <a href="https://github.com/James-Benson"><i class="fab fa-github"></i></a>
+            <a href="#"><i class="fab fa-facebook-square"></i></a>
+            <a href="#"><i class="fab fa-twitter-square"></i></a>
+        </nav>
+    </header>
+    <main>
+        About
```

Comment:
id consider putting your content into a box with a semi-transparent dark background, so it stands out from the image a bit more